packages/amplify-cli/src/snapshot-serializer.ts

```diff
@@ -0,0 +1,12 @@
+const originalPlatform = process.platform;
```

Comment:
This file should go in `src/__tests__`. The main `src` directory is for application code.